### PR TITLE
feat:Enabled state transitions without modification of models

### PR DESCRIPTION
**Pull request by shijinrajbosch**
## Description

Enabled state transitions without modification of models.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.2.16
+### Added
+- Enabled state transitions without modification of models.
+
+### Fixed
+=======
```
Comment (tunacicek) [inline CHANGELOG.md:12]: please remove this line

Comment (shijinrajbosch) [reply CHANGELOG.md]: > please remove this line

Thanks for the review comment @tunacicek . 

I have removed the line.

**Comment by tunacicek** [comment]
@shijinrajbosch : could you resolve the merge conflicts.

**Comment by shijinrajbosch** [comment]
> @shijinrajbosch : could you resolve the merge conflicts.

Hi @tunacicek , I have resolved the merge conflicts.

**Review by tunacicek — APPROVED** [review]
LGTM

**Pull request merged**